client/state/posts/selectors.js

```diff
@@ -359,6 +360,29 @@ export const isEditedPostDirty = createSelector(
 );
 
 /**
+  * Returns true if the edited post has content
+  * (title, excerpt or content not empty)
+  *
+  * @param  {Object}  state  Global state tree
+  * @param  {Number}  siteId Site ID
+  * @param  {Number}  postId Post ID
+  * @return {Boolean}        Whether the edited post has content or not
+ */
+export const editedPostHasContent = createSelector(
+	( state, siteId, postId ) => {
+		const editedPost = getEditedPost( state, siteId, postId );
+		return (
+			!! editedPost &&
+			(
+				some( [ 'title', 'excerpt' ], ( field ) => editedPost[ field ] && !! editedPost[ field ].trim() ) ||
+				! isPostContentEmpty( editedPost.content )
+			)
+		);
+	},
+	( state ) => [ state.posts.items, state.posts.edits ]
+);
```

Comment:
Maybe it's me, but I don't see much justification to create a memorized selector here. Is `getEditorPost()` really heavy or something? I don't see the dependence on `state.posts.items` or `state.posts.edits`.

Sorry for bringing it up again, but it stands out and bothers me - feels confusing.